packages/digitalproperty-app/lib/cmds/landregistry_cmds/addTitles.js

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env node
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+const LandRegistry = require('./../../landRegistry.js');
+const winston = require('winston');
+const LOG = winston.loggers.get('application');
+
+
+
+
+
+module.exports.command = 'addTitle [options]';
+module.exports.desc = 'Add a set of land titles';
+module.exports.builder = function(yargs) {
+
+    return yargs
+    .option('f', { alias: 'firstName', describe: 'Add your first name to the registry.', type: 'string'})
+    .option('l', { alias: 'lastName', describe: 'Add your last name to the registry.', type: 'string'})
+    .implies('firstName','lastName')
```

Comment:
do we need an `implies('lastname','firstname') so you need both? 

If we need both probably eaiser to put in a required:true to both of them instead of implies.